modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -17,13 +17,16 @@
 [NOTE]
```

Comment:
Let's decide what to call these - snippets or fragment, then change the title to one or the other (Text snippets for example). Then add a definition at the start of the module:

A text snippet is text that is stored in an asciidoc file that you can include in a module. Text snippets usually contain text that is repeated in more than one module. Using a text snippet insures consistency across modules. Text snippets can be used for the following types of information:

Reply:
I vote for snippets! Is there any argument for fragment?